src/docs/product/user-feedback/index.mdx

```diff
@@ -1,25 +1,76 @@
 
-![User Feedback page with submissions.](user-feedback.png)
+For example, using Session Replay with the User Feedback Widget could save you time, helping you quickly understand the context of the user problem by seeing exactly what they experienced before submitting their feedback.
 
-In each submission, you can click the “View Event” link, which takes you to the **Issue Details** page for that error event (or instance of the issue). From there, you can dig into what caused the issue and start working to resolve it:
+## User Feedback Widget
 
-![The event that generated the user feedback on the Issue Details page.](user-feedback-event.png)
+The User Feedback Widget is an out-of-the-box, persistent widget you can add anywhere on your web application. It allows users to submit feedback quickly and easily any time they encounter something that isn’t working as expected. Learn more about how to install and customize it [here](https://github.com/getsentry/sentry-javascript/tree/develop/packages/feedback#setup).
+
+![User Feedback widget.](user-feedback-widget.png)
+
+Feedback collected via the Feedback Widget may have some or all of the following information (depending on what you mark as required and what’s available from the error context):
+
+- Description from the user
+- User email
+- Associated replay event (with up to 60 seconds of replay recording before the user submitted the feedback)
+- Learn more about how to set up Session Replay [here](/product/session-replay/getting-started/).
+- URL of the page the user was on when they submitted the feedback
+- Tags
+
+## Crash-Report User Feedback
+
+Another way to collect feedback is by installing the Crash-Report User Feedback modal. This allows users to submit feedback after they experience an error via an automatic modal that pops up after an error occurs. Learn more about how to install and customize it [here](https://docs.sentry.io/platforms/javascript/enriching-events/user-feedback/#embeddable-javascript-widget).
+
+![Crash-Report User Feedback.](crash-report-user-feedback.png)
+
+Feedback collected via the Crash-Report modal will have the following information:
+
+- Description from the user
+- Sentry Issue preview and link to the Issue Details page
+- URL of the page the user was on when they submitted the feedback
+- Tags
+
+## Where to View User Feedback
+
+Whether you choose to collect user feedback via a Feedback Widget, a Crash-Report modal, or both, you’ll be able to view it from the **User Feedback** page. The page is chronologically-organized with the most recent, unresolved feedback at the top. You can click on individual feedback to see more details all in one view, similar to your e-mail inbox. The right-hand side shows the **User Feedback Details** view, which includes the user’s description and any available debugging context.
```

Comment:
Are there any differentiators between feedback from the form/widget and crash report in the UI? Or any way to filter for one or the other? If so, seems worthwhile to include that info.

Reply:
Great point! There are differences indeed, lets expand

Reply:
We could add to the bottom (after the last sentence in this heading:

> Feedback collected via the Crash-Report Modal displays a _crash-report_ label. And a link to the issue that resulted in the modal popping up.

Or something?

Reply:
@bruno-garcia ^ I like this!

Reply:
Yes, I like that too! 

> Feedback collected via the Crash-Report Modal displays a _crash-report_ label and a link to the issue that resulted in the modal popping up.

Would just make it one sentence instead of two.

But I'm not sure if I'm seeing the label correctly, as least in the demo app. It looks like this feedback has a linked issue but I don't see the label. Can you verify what it's supposed to look like?
https://demo.sentry.io/feedback/?feedbackSlug=android%3A4657517631&referrer=feedback_list_page&statsPeriod=7d